plugin.py

```diff
@@ -14,3 +19,25 @@ class LspVolarPlugin(NpmClientHandler):
     package_name = __package__
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', '@volar', 'server', 'out', 'index.js')
+
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        window: sublime.Window,
+        initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        configuration: Optional[ClientConfig] = None
+    ):
+        if not workspace_folders or not configuration:
+            return
+        ts_path = ''
+        resolve_module_script = os.path.join(cls._server_directory_path(), 'resolve_module.js')
+        first_folder = workspace_folders[0].path
+        proc = subprocess.Popen(['node', resolve_module_script, first_folder, 'typescript/lib/tsserverlibrary.js'], stdout=subprocess.PIPE)
```

Comment:
Instead of `node` use `cls._node_bin()`.

Reply:
Would it work to use higher-level API `subprocess.check_output` (with `universal_newlines=True`)?

Reply:
Thanks.
Yes, it works.